xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/suggest/xwiki.selectize.js

```diff
@@ -203,27 +203,21 @@ define('xwiki-selectize', [
         });
       }
     }
-    let oldOnPositionDropdown = this.selectize.onOptionSelect;
-    this.selectize.onOptionSelect = function(e) {
-      oldOnPositionDropdown.call(this, e);
+    let oldOnOptionSelect = this.selectize.onOptionSelect;
+    this.selectize.onOptionSelect = function(...args) {
+      const result = oldOnOptionSelect.apply(this, args);
+      // clear aria-selected state on all previously selected elements first
       this.get$('dropdown').find('.option').attr('aria-selected','false');
-      // clear selection on all previously selected elements first
+      // then set the aria-selected state of the newly selected element.
       this.get$('dropdown').find('.selected').attr('aria-selected','true');
+      return result;
     }
     let oldSetActiveOption = this.selectize.setActiveOption;
-    this.selectize.setActiveOption = function(option, scroll, animate) {
-      if (this.liveRegion) {
-        if (option instanceof jQuery) {
-          this.liveRegion.text(option.text());
-        } else if (option instanceof HTMLElement) {
-          this.liveRegion.text(option.innerText);
-        } else {
-          this.liveRegion.text("");
-        }
-      }
-      oldSetActiveOption.call(this, option, scroll, animate);
+    this.selectize.setActiveOption = function(option, ...args) {
+      this.liveRegion?.text($(option).text());
+      oldSetActiveOption.call(this, option, ...args);
```

Comment:
```suggestion
      return oldSetActiveOption.call(this, option, ...args);
```

In case the original ``setActiveOption`` is modified to return some value (e.g. the previously active option), in future versions of Selectize.

Reply:
Okay! Better safe than sorry :)